src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -284,6 +297,18 @@ public void setAuthorList(List<Author> authorList) {
         });
     }
 
+    public void addAuthorList(List<Author> authorList) {
```

Comment:
Let's rename this to `addAuthors`.
The current name sounds like it is adding a `List<Author>` into a `Collections<List<Author>>`, when we are actually appending all the authors in the `authorList` to the `RepoConfiguration` own author list.
Also, add header comments for the method.